Source/WebCore/PAL/pal/spi/mac/NSMenuSPI.h

```diff
@@ -35,6 +35,11 @@
 
 + (NSMenuItem *)standardWritingToolsMenuItem;
 
+#if ENABLE(CONTEXT_MENU_IMAGES_FOR_INTERNAL_CLIENTS)
++ (NSString *)_systemImageNameForAction:(SEL)action;
+@property (strong, setter=_setActionImage:) NSImage *_actionImage;
```

Comment:
I think these probably need to go into their own staging category? Rather than being related to rdar://129192954 (WritingTools)

```
#if ENABLE(CONTEXT_MENU_IMAGES_FOR_INTERNAL_CLIENTS)

@interface NSMenuItem (Staging_138651669)
+ (NSString *)_systemImageNameForAction:(SEL)action;
@property (strong, setter=_setActionImage:) NSImage *_actionImage;
@end

#endif
```